lib/api.js

```diff
@@ -37,7 +38,11 @@ function open(options = {}) {
   }
 
   return Promise
-          .resolve(fetch(uri, opts))
+          .try(() => {
+            return retry(() => {
+              return fetch(uri, opts);
+            }, { max_tries: 5, interval: 1000 });
```

Comment:
change from:
`{ max_tries: 5, interval: 1000 }`

to:
`{ max_tries: 10, interval: 500, backoff: 2, max_interval: 4000 }`

on all of them